appcues/src/main/java/com/appcues/ElementTargetingStrategy.kt

```diff
@@ -97,6 +97,11 @@ public data class ViewElement(
      */
     val type: String,
 
+    /**
+     * A value representing the display name of the view.
+     */
+    val displayName: String?,
```

Comment:
This is exactly where I would expect it (not in the selector itself, but part of the ViewElement)

Currently, the expectation is that this is only required to be set if the view has a selector - so many views may not have a display name and that is OK. Pondering if we should make `displayName` a required prop on the `ElementSelector` interface. Considering cross platform, like React Native, where it has it's own implementation of ElementSelector and it's own targeting strategy completely - it will need updated as well to provide a displayName.

I think it's probably best to _not_ require it on `ElementSelector` and leave it up to each implementation to handle as desired. Which is in line with what is here, just thinking it through.

Reply:
I think it makes sense to leave displayName here because it would be part of the interface contract the builder would expect to display some "proper name", this is in line with open builder concept right, this is why I asked about the type, right now type is not used for anything but we could enhance it and use that as a parameter to know which icons to show in builder, etc.